ui/components/accounts/AccountsList.vue

```diff
@@ -37,6 +46,16 @@ export default {
       type: Array,
       required: true
     }
+  },
+  computed: {
+    ...mapGetters('Accounts', ['areAllAccountsSelected'])
+  },
+  methods: {
+    ...mapActions('Accounts', ['toggleSelectionAll']),
+
+    toggleAllAccountsSelection () {
```

Comment:
I don't see a reason for introducing a wrapping function. Could call the `toggleSelectionAll` directly on `@change` in the template. Please clarify if needed.